cypress/fixtures/docs/validations.md

```diff
@@ -421,6 +422,86 @@ The 'schemaOnly` validation type allows to skip content validation and stops at
 
 ```
 
+## screenshotValidation
+FCS supports screenshot validation. This validation is used to validate the screenshot captured by the platform. The screenshot validation object should be added to the validation objects.
+
+ - The request override function must be added to the configModule to capture the screenshot based on the platform.
+ - The config module is responsible for capturing the screenshot and returning the validation response back to FCS.
+ - `screenshot` is the function name which should be added in `requestModules/fcs.js` file and the same function can be defined in `responseModules/fcs.js` file to capture the screenshot response and do validations. 
+ - The screenshot validation response received from the config module looks like the format below.
+    ```javascript
+    {
+      "status": "pass/fail",
+      "validations": [
+        {"status":"pass/fail"},
+        {"status":"pass/fail"}
+      ]
+    }
+    ```
+    In the above response, `status` is the key which indicates the overall status of the screenshot validation. `validations` is the array of objects which holds the status of each validation object. `status` is the required field to determine the status of the validation.
+
+### Validation object format: 
+
+```
+{
+  "data": [
+    {
+      "type": "screenshotValidation",
+      "validations": []
+    }
+  ]
+}
+
+```
+
+### Params:
+
+| **Param**   | **Type** | **Description**                                                            |
+| ----------- | -------- | -------------------------------------------------------------------------- |
+| data        | array    | An array that holds the entire set of validation objects.                  |
+| type        | string   | The value that indicates the type of validation.                           |
+| validations | array    | The array that holds all the data for validation, like value, format, etc. |
```

Comment:
why is there no mention about the screenshot validation flag ?

Reply:
added the comment for enableScreenshots flag